docs/theming.md

```diff
@@ -185,9 +185,174 @@ extension MyViewController {
 ```
 <!--</div>-->
 
+## Migration guide: Themers to Theming extensions
+
+This migration guide covers the typical migration path from Themer usage to Theming extensions.
+
+Theming extensions are discussed in detail above. Themers are discussed in the [Themers](#themers)
+section. Themers will eventually be deprecated and deleted. Theming extensions are the recommended
+replacement.
+
+In general, every component's `Themer` targets will gradually be replaced by a single `Theming`
+extension target. This includes:
+
+- ColorThemer
+- FontThemer
+- ShapeThemer
+- TypographyThemer
+- Component Themers (e.g. CardThemer)
```

Comment:
Tip: Nit: "Component Themer**s**" should be singular, since there is only one of them per component. I may be wrong.

Reply:
Buttons has multiple component themers >.<

Reply:
Ack.